end_to_end/1-data-prep-e2e.ipynb

```diff
@@ -0,0 +1,861 @@
+{
```

Comment:
<p>The link to the video does not work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>